modules/support-log-gather-config-params.adoc

```diff
@@ -0,0 +1,85 @@
+
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="support-log-gather-config-params_{context}"]
+= Configuration parameters for MustGather custom resource
+
+You can manage your `MustGather` CR by creating a YAML file that specifies the parameters for the data collection and upload process.
+The following table provides an overview of the parameters that you can configure in the `MustGather` CR.
+
+[cols="1,3", options="header"]
+|===
+|Field Name |Description
+
+|`spec.serviceAccountRef`
+|Optional. Defines the service account used to run the must-gather job pod. The default value is `+kubebuilder:default:="{Name:default}"`.
+
+|`spec.serviceAccountRef.name`
+|Optional. Specifies the name of the service account.
+
+|`spec.audit`
+|Optional. Specifies whether to collect audit logs. The valid values are `true` and `false`. The default value is `false`.
+
+|`spec.mustGatherTimeout`
+|Optional. Specifies the time limit for the `must-gather` command to complete. The value must be a floating-point number with a time unit. The valid units are `s` (seconds), `m` (minutes), `h` (hours), or `d` (days). By default, no time is limit set.
+
+|`spec.proxyConfig`
+|Optional. Defines the proxy configuration to be used. The default value is set to the cluster-level proxy configuration.
+
+|`spec.proxyConfig.httpProxy`
+|Specifies the URL of the proxy for HTTP requests.
+
+|`spec.proxyConfig.httpsProxy`
+|Specifies the URL of the proxy for HTTPS requests.
+
+|`spec.proxyConfig.noProxy`
+|Specifies a comma-separated list of domains for which the proxy must not be used.
+
+|`spec.uploadTarget`
+|Optional. Defines the upload location for the `must-gather` bundle. By default, the bundle is uploaded to `sftp.access.redhat.com`.
+
+|`spec.uploadTarget.type`
+|Specifies the type of upload location for the `must-gather` bundle. The only supported value is `SFTP`.
+
+|`spec.uploadTarget.sftp.caseID`
+|Specifies the Red{nbsp}Hat Support case ID for which the diagnostic data is collected.
+
+|`spec.uploadTarget.sftp.caseManagementAccountSecretRef`
+|Defines the credentials required for authenticating and uploading the files to the Red{nbsp}Hat Customer Portal support case. The value must contain a `username` and `password` field.
+
+|`spec.uploadTarget.sftp.caseManagementAccountSecretRef.name`
+|Specifies the name of the Kubernetes secret that contains the credentials.
+
+|`spec.uploadTarget.sftp.internalUser`
+|Optional. Specifies whether the user provided in the `caseManagementAccountSecretRef` is a Red{nbsp}Hat internal user. The valid values are `true` and `false`. The default value is `false`.
```

Comment:
@KeenonLee If the other comment is valid, we might have to remove it from here too?